tools/package-builder/src/core/package-info.ts

```diff
@@ -680,7 +683,26 @@ const readPackageJson = (filePath: string) => parsePackageJson(fs.readFileSync(f
 function parsePackageJson(data: string) {
   const parsedData = JSON.parse(data);
   // TODO: try/catch and transform errors on human-readable format
-  return packageJsonSchema.parse(parsedData) as packageJson;
+  const result = packageJsonSchema.safeParse(parsedData);
+  if (result.success) {
+    return result.data as packageJson;
+  }
+
+  const err = JSON.stringify(result.error.format(), null, 2);
+  if (err.includes(`Unrecognized key(s) in object: 'cmd'`)) {
+    throw new Error(`
+    Error: "cmd" was renamed to "command" in package.json in block-software.entrypoints.
+    The syntax has changed: all curly braces should be doubled, for example:
+        {pkg} -> {{pkg}}
+
+    The new "command" field supports rendering of such arguments, for example:
```

Comment:
> The new "command" field supports rendering of such arguments, for example:

Let's say not rendering, but clearly point to math expressions and environment variables
> The new "command" field supports math expressions and env variables resolution, for example: